contents/blog/why-we-raised-a-15m-series-b-ahead-of-schedule.md

```diff
@@ -0,0 +1,27 @@
+---
+date: 2021-06-14
+title: Why we raised a $15m Series B ahead of schedule
+author: joe-martin
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+By: [Joe Martin](https://www.linkedin.com/in/joemartinwords/), Product Marketing
```

Comment:
Since we have a blog author field now, you can add yourself as the author without putting it in the post body!

1. Remove this line
2. Below `hideAnchor: true` on line 8, add `author: joe-martin` (since you have an [author profile](https://github.com/PostHog/posthog.com/blob/master/contents/authors.md) already)

(I just added some instructions about this to our [Marketing -> Blog handbook entry](https://github.com/PostHog/posthog.com/pull/1477/files).)

Reply:
Awesome, thanks! 


@shehuphd Would be worth updating the notes you gave me with this, please